pkg/egressgateway/manager_privileged_test.go

```diff
@@ -61,11 +61,13 @@ var (
 
 	identityAllocator = testidentity.NewMockIdentityAllocator(nil)
 
-	nodeGroup1Labels = map[string]string{"label1": "1"}
-	nodeGroup2Labels = map[string]string{"label2": "2"}
+	nodeGroupNotFoundLabels = map[string]string{"label1": "notfound"}
+	nodeGroup1Labels        = map[string]string{"label1": "1"}
+	nodeGroup2Labels        = map[string]string{"label2": "2"}
 
-	nodeGroup1Selector = &slimv1.LabelSelector{MatchLabels: nodeGroup1Labels}
-	nodeGroup2Selector = &slimv1.LabelSelector{MatchLabels: nodeGroup2Labels}
+	nodeGroupNotFoundSelector = &slimv1.LabelSelector{MatchLabels: nodeGroupNotFoundLabels}
```

Comment:
I think you dropped a test here that could have been kept AFAICT (or if too cumbersome to drop those change to the labels/labels selectors should probably go as well).

Reply:
I dropped those changes as they were related to the excluded CIDRs test cases, I'll drop also the unused variables to make the linter happy :+1: